.github/workflows/feature_deploy.yml

```diff
@@ -70,7 +70,8 @@ jobs:
           username: ${{ env.acr_username }}
           password: ${{ env.acr_password }}
           repository: ${{ matrix.repository }}
-          path: ${{ matrix.path || matrix.repository }}
+          context: .
+          dockerfile: ${{ matrix.path || matrix.repository }}/Dockerfile
```

Comment:
This is the main change, with everything else just being auto-formatting.

* use `path` if you want to change both the location of the Dockerfile and context to run it in
* use `context` and `dockerfile` if you want to set the context to run to a different location to the Dockerfile

A successful deployment to the feature environment with this change can be seen [here](https://github.com/UK-Export-Finance/dtfs2/actions/runs/7901475398).

Reply:
what's the difference between matrix.path and matrix.repository? when is one not defined and we need to fallback to the other?

Reply:
I think this is to handle the case where the `portal-ui` project (name in package.json) is different from the directory name of `portal`. The path for this is specified on lines 59-61 above:
```
include:
  - path: portal
     repository: portal-ui
```

Reply:
ahh gotcha, makes sense